source/lmp/fix_dplr.cpp

```diff
@@ -115,6 +115,8 @@ FixDPLR::FixDPLR(LAMMPS *lmp, int narg, char **arg)
   for (int ii = 0; ii < sel_type.size(); ++ii) {
     dpl_type.push_back(type_asso[sel_type[ii]]);
   }
+  dpl_type_sorted = dpl_type;
```

Comment:
Do we need to store two vectors? It seems to me that `dpl_type` is never used other than this line.

Reply:
Solved by commit [9cbc513](https://github.com/deepmodeling/deepmd-kit/pull/2580/commits/9cbc51338b9d9f4412b100d6faa85ffc5a84269c).